packages/client/src/workflow-client.ts

```diff
@@ -880,19 +1099,44 @@ export class WorkflowClient extends BaseClient {
         }
         return temporal.api.history.v1.History.create({ events });
       },
-      async signal<Args extends any[]>(def: SignalDefinition<Args> | string, ...args: Args): Promise<void> {
+      async startUpdate<Ret, Args extends unknown[]>(
```

Comment:
> Missing a `getUpdateHandle` method

That was intentional; it was my understanding we were not aiming to include that in the initial release. Python does not have it, but I see .NET does. Let's add in a separate PR -- I think it will make review easier and generally safer than squeezing into mostly a reviewed PR.

Reply:
I'm okay adding it later.